sdk/core/azure-core/src/main/java/com/azure/core/cryptography/KeyEncryptionKeyResolver.java

```diff
@@ -14,5 +14,5 @@
      * @param keyId The key identifier of the key encryption key to retrieve
      * @return The key encryption key corresponding to the specified {@code keyId}
      */
-    KeyEncryptionKey resolveKey(String keyId);
+    KeyEncryptionKey buildKeyEncryptionKey(String keyId);
```

Comment:
Same here - use a different prefix other than `build`

Reply:
as discussed not an issue